packages/api-types/out/schema.ts

```diff
@@ -0,0 +1,1708 @@
+  };
+  '/api/praise/forward': {
+    /** Forward praise item */
+    post: operations['PraiseController_forward'];
+  };
+  '/api/quantifications/export': {
+    /** Exports quantifications document to json or csv. */
+    get: operations['QuantificationsController_export'];
+  };
+  '/api/useraccounts': {
+    /** Get UserAccount by UserId our AccountId. */
+    get: operations['UserAccountsController_findOne'];
+    /** Create a UserAccount */
+    post: operations['UserAccountsController_create'];
+    /** Update UserAccount by UserId or AccountId */
+    patch: operations['UserAccountsController_update'];
+  };
+  '/api/useraccounts/export': {
+    /** Exports UserAccounts document to json or csv. */
+    get: operations['UserAccountsController_export'];
+  };
+  '/api/api-key': {
+    /** List all API keys */
+    get: operations['ApiKeyController_findAll'];
+    /** Create API key */
+    post: operations['ApiKeyController_createApiKey'];
+  };
+  '/api/api-key/{id}': {
+    /** Get API key by ID */
+    get: operations['ApiKeyController_findOne'];
+    /** Update API key description */
+    put: operations['ApiKeyController_updateApiKeyDescription'];
+    /** Revoke API key */
+    delete: operations['ApiKeyController_revokeApiKey'];
+  };
+  '/api/auth/eth-signature/nonce': {
+    /** Generates a nonce for the user and returns it */
+    post: operations['AuthController_nonce'];
+  };
+  '/api/auth/eth-signature/login': {
+    /** Verifies a user's signature and returns a JWT token */
+    post: operations['AuthController_login'];
+  };
+}
+
+export type webhooks = Record<string, never>;
+
+export interface components {
+  schemas: {
+    ActivateInputDto: {
+      /** @example 0xAAB27b150451726EC7738aa1d0A94505c8729bd1 */
+      identityEthAddress: string;
+      /** @example darth#3455 */
+      accountId: string;
+      /** @example 0xdb4bb91357b23083ec2a36dc1fe23e59b71434fc020542da7e983df206ed06611e275eb30e239508f9758c0608dca6cef5619c41b50a48f22bdb36a8dabc2d201c */
+      signature: string;
+    };
+    UserAccountNoUserId: {
+      /** @example 63b428f7d9ca4f6ff5370d05 */
+      _id: string;
+      /** @example 63b428f7d9ca4f6ff5370d05 */
+      user: string;
+      /** @example 098098098098098 */
+      accountId: string;
+      /** @example darth#6755 */
+      name: string;
+      /** @example 098098098087097 */
+      avatarId: string;
+      /** @example DISCORD */
+      platform: string;
+      /** Format: date-time */
+      createdAt: string;
+      /** Format: date-time */
+      updatedAt: string;
+    };
+    User: {
+      /** @example 5f9f1b9b9b9b9b9b9b9b9b9b */
+      _id: string;
+      /** @example 0xAAB27b150451726EC7738aa1d0A94505c8729bd1 */
+      identityEthAddress: string;
+      /** @example 0xAAB27b150451726EC7738aa1d0A94505c8729bd1 */
+      rewardsEthAddress: string;
+      /** @example darth */
+      username: string;
+      /** @example ["USER"] */
+      roles: string;
```

Comment:
Hmm, this becomes string instead of string[] even though api2 schema looks correct